client/lib/performance-tracking/components/performance-tracker-stop.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { stop } from '@automattic/browser-data-collector';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import { getSectionName } from 'state/ui/selectors';
+
+const PerformanceTrackerStop = ( { sectionName } ) => {
+	useEffect( () => {
+		if ( config.isEnabled( 'rum-tracking/logstash' ) ) {
+			stop( sectionName );
+		}
+	}, [ sectionName ] );
```

Comment:
You could use the hook here instead of writing the same `useEffect` call again.

Reply:
Done